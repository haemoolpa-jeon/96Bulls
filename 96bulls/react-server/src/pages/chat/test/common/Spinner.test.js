import React from 'react';
import Spiiner from '../../components/common/Spinner';
import {shallow} from 'enzyme/build';

describe('Spinner', () => {
  let component = null;
  it('초기 렌더링이 문제없이 되야함', () => {
    component = shallow(<Spiiner/>);
  });

  it('초기 렌더링 스냅샷 일치함', () => {
    expect(component).toMatchSnapshot();
  });
});
