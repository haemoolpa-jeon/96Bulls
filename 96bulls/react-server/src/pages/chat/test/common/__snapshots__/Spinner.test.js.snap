// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner 초기 렌더링 스냅샷 일치함 1`] = `
<div
  className="dimmer"
>
  <onlyUpdateForKeys(Loader)
    className=""
    color="#28b485"
    loading={true}
    size={60}
    sizeUnit="px"
  />
</div>
`;
